corehq/apps/reports/standard/deployments.py

```diff
@@ -139,6 +141,22 @@ def rows(self):
             )
         return rows
 
+    @property
+    def export_table(self):
+        def _fmt_ordinal(ordinal):
+            if ordinal >= 0:
```

Comment:
does this need to check for `None` or is it guaranteed to be a number by this time?